equalsverifier-core/src/test/java/nl/jqno/equalsverifier/integration/operational/OriginalStateTest.java

```diff
@@ -53,7 +53,7 @@ void allValuesReturnToOriginalState_whenEqualsVerifierIsFinishedWithException()
         }
         catch (AssertionError e) {
             // Make sure EV fails on a check that actually mutates fields.
-            assertThat(e.getMessage().contains("Mutability")).isTrue();
+            assertThat(e.getMessage()).contains("Mutability");
```

Comment:
The assertion failure here would go from expected true was false to expected a string containing Mutability, but was ....